etl/steps/grapher/owid/latest/key_indicators.py

```diff
@@ -0,0 +1,107 @@
+from copy import deepcopy
+from typing import Any, List
+
+import numpy as np
+from owid import catalog, walden
+
+from etl import grapher_helpers as gh
+from etl.paths import DATA_DIR
+
+KEY_INDICATORS_GARDEN = DATA_DIR / "garden/owid/latest/key_indicators"
+YEAR_THRESHOLD = 2022
+
+
+def run(dest_dir: str) -> None:
+    # NOTE: this generates shortName `population_density__owid_latest`, perhaps we should keep it as `population_density`
+    # and create unique constraint on (shortName, version, namespace) instead of just (shortName, namespace)
+    garden_dataset = catalog.Dataset(KEY_INDICATORS_GARDEN)
+    dataset = _adapt_dataset_metadata_for_grapher_patch(dest_dir, garden_dataset)
+
+    # Get population table
+    table = garden_dataset["population"].reset_index()
+    # Create population new metrics
+    table = _split_in_projection_and_historical(table, YEAR_THRESHOLD, "population")
+    table = _split_in_projection_and_historical(table, YEAR_THRESHOLD, "world_pop_share")
+    # table["population_historical"] = deepcopy(table["population"])
+    # table["population_projection"] = deepcopy(table["population"])
+    # Add population table to dataset
+    table = _adapt_table_for_grapher_patch(table)
+    dataset.add(table)
+
+    # Add land area table to dataset
+    table = _adapt_table_for_grapher_patch(garden_dataset["land_area"].reset_index())
+    dataset.add(table)
+
+    # Add population density table to dataset
+    table = _adapt_table_for_grapher_patch(garden_dataset["population_density"].reset_index())
+    dataset.add(table)
+
+    # Save dataset
+    dataset.save()
+
+
+def _adapt_dataset_metadata_for_grapher_patch(dest_dir: str, garden_dataset: walden.Dataset) -> catalog.Dataset:
+    """Fixes source separator in dataset ' ; ' -> '; '"""
```

Comment:
@pabloarosado is this something we can do for all datasets in `combine_metadata_sources` or do you have any objections?

Reply:
@lucasrodes since this just got merged, could you remove those `_patch` functions and then merge your PR? (I have another PR that is waiting for this one)

Reply:
On it! Will ping you